lib/auth/clt.go

```diff
@@ -1312,7 +1312,27 @@ func (c *Client) UpsertPassword(user string, password []byte) error {
 	return nil
 }
 
-// UpsertUser user updates or inserts user entry
+// CreateUser inserts a new entry in a backend.
+func (c *Client) CreateUser(ctx context.Context, user services.User) error {
+	clt, err := c.grpc()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	userV2, ok := user.(*services.UserV2)
+	if !ok {
+		return trace.BadParameter("unsupported user type %T", user)
+	}
+
+	_, err = clt.CreateUser(ctx, userV2)
+	if err != nil {
```

Comment:
Prefer combining the call with error check where possible:
```go
if _, err := clt.CreateUser(ctx, userV2); err != nil {
...
}
```

This makes `err` local to the if scope and helps avoid subtle shadowing bugs.

Reply:
revised